src/test/java/com/eliotlash/molang/FunctionsTest.java

```diff
@@ -91,11 +91,14 @@ void testMiscFunctions() throws Exception {
         assertEquals(10.0, evaluate("math.abs(-10)"));
         assertEquals(10.0, evaluate("math.abs(10)"));
 
-        assertEquals(Math.E, evaluate("math.exp(1)"));
+        assertEquals( 2.7182818284590455, evaluate("math.exp(1)"));
```

Comment:
? lol

Reply:
There is a discrepancy in the floating point accuracy between the constant `Math.E` & the implementation of `Math.exp(1)` on some JVM implementations, at least on macOS.


Constant E: `2.718281828459045`


Results on OpenJDK & Zulu 8, 16, & 17, and IntelliJs version of JDK (v11), the result of `Math.exp(1)` == `2.7182818284590455`

Results on GraalVM 17: `2.718281828459045`


Given that this seems to be up to the implementation of the JDK itself and that the result is not reliably reproducible across multiple versions of the JDK, I think we should just remove this assert.

Reply:
I would just use `math.exp(1)` as it will give the same result

Reply:
Good point! I switched to using `Math.exp(1)`, and tried resolved merge conflicts